tools/fpgaconf/fpgaconf.c

```diff
@@ -389,10 +390,21 @@ int read_bitstream(char *filename, struct bitstream_info *info)
 	FILE *f;
 	long len;
 	int ret;
+	struct stat file_mode = { 0 };;
```

Comment:
May want to memset file_mode to 0 instead of using the initializer syntax.  Klocwork is very picky about that.

Reply:
And you only need one ';'. We have a limited supply. ;)